pkg/windows/service.go

```diff
@@ -10,16 +10,19 @@ type service struct {
 	name string
 	// args is the arguments that the binary will be ran with
 	args string
+	// dependencies is a list of the services that this service is dependant on
```

Comment:
nit: `...list of the services' names...` since service is a struct
nit: `dependant` --> `dependent`